### PR TITLE
Add delegate value to inscription endpoint

### PR DESCRIPTION
**Pull request by phorkish**
Add the delegate ID to the `/inscription/[id]` endpoint. If there is no delegate then the response doesn't include that field.

### CODE REVIEW

**Comment by elocremarc** [comment]
@gmart7t2 This PR is attempting the same thing as yours gonna link these together. 
https://github.com/ordinals/ord/pull/3761

File: src/api.rs [hunk 1]
```diff
@@ -103,6 +103,8 @@ pub struct InscriptionRecursive {
   pub charms: Vec<Charm>,
   pub content_type: Option<String>,
   pub content_length: Option<usize>,
+  #[serde(skip_serializing_if = "Option::is_none")]
```
Comment (raphjaph) [inline src/api.rs:106]:
```suggestion
```
This is a good idea but in order to stay consistent with the other Option fields we should remove it

**Review by raphjaph — CHANGES_REQUESTED** [review]
LGTM, just a small change

**Comment by gmart7t2** [comment]
@phorkish That "this branch is out-of-date with the base branch" means that your PR no longer merges cleanly to the upstream repo because something incompatible was merged. You need to merge your change to the master branch and push the change to your branch. 

It's kind of annoying to have to keep doing that, but someone needs to do it.

**Comment by raphjaph** [comment]
> @phorkish That "this branch is out-of-date with the base branch" means that your PR no longer merges cleanly to the upstream repo because something incompatible was merged. You need to merge your change to the master branch and push the change to your branch. 
> 
> It's kind of annoying to have to keep doing that, but someone needs to do it.

There's a feature on GitHub which auto updates branches if they can be merged cleanly. I'll try to find it and activate

**Comment by raphjaph** [comment]
#3784

**Review by raphjaph — CHANGES_REQUESTED** [review]
Sorry, forgot to add the we should add this field to to docs. Could you add the `delegate:  None` field to the examples in the `recursion.md` docs?

**Comment by phorkish** [comment]
Yep, done. Sorry about missing that.

**Review by raphjaph — APPROVED** [review]
LGTM

**Pull request merged**